<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditingTest</class>
 <widget class="QWidget" name="EditingTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="proverka_batton">
   <property name="geometry">
    <rect>
     <x>678</x>
     <y>10</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>teachWindow</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="questionEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>221</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QRadioButton" name="oneTrueAnswerButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>один правильный ответ</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="severalTruAnswerButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Несколько правильных ответов</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createAnswerButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>создать ответ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantityAnswerEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteAnswerButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>удалить ответ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberAnswerEdit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveQuestionButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>сохранить вопрос</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantityQuestionEdit">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>130</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveTestButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>сохранить тест</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openTestButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>550</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>открыть тест</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseQuestionButton">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>450</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>выбрать вопрос</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="chooseQuestionEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="createQuestionButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>создать вопрос</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameTestEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>520</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>колличество вопросов</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteQuestionButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>удалить вопрос</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
